src/zenon.cpp

```diff
@@ -269,6 +269,9 @@ void zenon::allocate_buffers()
     SUCCESS_OR_TERMINATE( zeMemAllocDevice( context, &memory_descriptor,
         alloc_size, 1, device, &im_buf6 ) );
 
+    hostDesc.flags = ZE_HOST_MEM_ALLOC_FLAG_BIAS_UNCACHED;
+    SUCCESS_OR_TERMINATE(zeMemAllocShared(context, &memory_descriptor, &hostDesc, alloc_size, 1, device, &sharedBuffer));
```

Comment:
shouldn't this be done only if user wants no blitter mode?

Reply:
done